src/main/java/com/hydratereminder/images/ImageCategories.java

```diff
@@ -0,0 +1,35 @@
+package com.hydratereminder.images;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+/**
+ * <p>Different categories for hydrate reminder images</p>
+ */
+@Getter
+@AllArgsConstructor
+public enum ImageCategories {
+    WATER("Water"),
+    TEA("Tea"),
+    ALCOHOL("Alcohol"),
+    DRINK("Drink"),
+    DRINKCONTAINERS("Drink Containers"),
+    FOOD("Food"),
+    POTIONCATEGORY("PotionCategory"),
```

Comment:
Can you change the string value to just "Potion"?

Reply:
I've changed it to "Potion"